templates/new/dodo.py.txt

```diff
@@ -63,3 +64,37 @@ def task_batch_local():
         'basename': 'batchLocal',
         'actions': ["%s -m %s --mode batch" % (sys.executable, PACKAGE_PATH)]
     }}
+
+def task_jupyter():
+    command = "rm -r work; pip install -r /home/jovyan/requirements.txt; /usr/local/bin/start.sh jupyter notebook --NotebookApp.token=''"
+    
+    def run_command():
+        process = subprocess.Popen([
+            "docker", 
+            "run",
+            "--rm",
+            "--name", "{project_name}_surround_notebook",
+            "--volume", "%s:/home/jovyan" % CONFIG['volume_path'], 
```

Comment:
Is there any reason why you are mounting to `/home/jovyan` rather than `/app`? It would be easier to diagnose issues in specific projects if the tasks were all consistent.

Reply:
Fixed